package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lens-platform-sdk",
-  "version": "9.2.1",
+  "version": "9.2.2",
```

Comment:
We don't need to manually add patch number, the CI should do it automatically when this PR is merged.